ddtrace/appsec/iast/_taint_tracking/TaintTracking/Source.cpp

```diff
@@ -14,13 +14,15 @@ Source::Source(string name, string value, OriginType origin)
   : name(move(name))
   , value(move(value))
   , origin(origin)
-{}
+{
+}
```

Comment:
Unintended changes?

Reply:
I believe this was a git hook actually

Reply:
Apparently this is related to using a different clang-format version (>=16) than in CI (clang-format==10)